packages/front-end/components/PersonalAccessTokens/PersonalAccessTokens.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { FC, useCallback, useMemo, useState } from "react";
+import { FaKey } from "react-icons/fa";
+import Link from "next/link";
+import { ApiKeyInterface, SecretApiKey } from "back-end/types/apikey";
+import { ApiKeysTable } from "@/components/ApiKeysTable/ApiKeysTable";
+import ApiKeysModal from "@/components/Settings/ApiKeysModal";
+import { useAuth } from "@/services/auth";
+import { groupApiKeysByType } from "@/services/secret-api-keys.utils";
+import useApi from "@/hooks/useApi";
+
+type PersonalAccessTokensProps = {
+  accessTokens: ApiKeyInterface[];
+  onDelete: (keyId: string | undefined) => () => Promise<void>;
+  onReveal: (keyId: string | undefined) => () => Promise<string>;
+  onCreate: () => void;
+};
+
+export const PersonalAccessTokens: FC<PersonalAccessTokensProps> = ({
+  accessTokens,
+  onDelete,
+  onReveal,
+  onCreate,
+}) => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <div>
+      {open && (
+        <ApiKeysModal
+          close={() => setOpen(false)}
+          onCreate={onCreate}
+          type="user"
+          secret={true}
+        />
+      )}
+
+      <div className="mb-4">
+        <h1>Personal Access Tokens</h1>
+        <p>
+          User keys have full read and write access to your user account.
+          Because of this, they must be kept secure and{" "}
+          <strong>must not be exposed to users</strong>.
```

Comment:
Is the use of `users` here confusing?  "user tokens must not be exposed to users"

Reply:
updated the copy to make it less redundant